pom.xml

```diff
@@ -96,12 +82,4 @@
             </plugin>
         </plugins>
     </build>
-
-    <distributionManagement>
-        <repository>
-            <id>github</id>
-            <name>PrivateStatus</name>
-            <url>https://maven.pkg.github.com/tinyoverflow/minecraft-privatestatus</url>
-        </repository>
-    </distributionManagement>
```

Comment:
Why do you remove this?

Reply:
I accidentally deleted it

Reply:
You can add again, editing the pull request